src/Administration/Resources/app/administration/src/app/component/meteor-wrapper/mt-text-editor/index.ts

```diff
@@ -21,15 +21,6 @@ Shopware.Component.register('mt-text-editor', {
     },
 
     props: {
-        /**
-         * @deprecated tag:v6.7.0 - The value prop is deprecated and will be replaced by modelValue
```

Comment:
can you please double check this case? I was not entirely sure what to do based on the deprecation, for now I just removed all references to value, not sure if we should simplify and also remove the whole `compatValue` as well 🤔

Reply:
@jleifeld

Reply:
It looks correct 👍

Reply:
@jleifeld seems that there were still some old usages with the value key, 
can you double check adjusting the usages as well, they are all in this commit: https://github.com/shopware/shopware/pull/6956/commits/53318c3e5ac95c2c9737027982d2f388f0ee9752

Reply:
Seem to be correct. The old usages were used because of the old editor compatibility. So this works 👍